src/test/java/ca/qaguru/oranghrmbatch27/tests/CustomersTests.java

```diff
@@ -0,0 +1,73 @@
+package ca.qaguru.oranghrmbatch27.tests;
+
+import ca.qaguru.oranghrmbatch27.library.TestBase;
+import ca.qaguru.oranghrmbatch27.pages.CustomersPage;
+import ca.qaguru.oranghrmbatch27.pages.LoginPage;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class CustomersTests extends TestBase {
+
+    //BATCH29-10
+    @Test
+    public void addNewCustomer()
+    {
+        LoginPage loginPage = new LoginPage(driver);
+        loginPage.login("Admin","admin123",true,null);
+        CustomersPage customersPage = new CustomersPage(driver);
+        customersPage.navigateToCustomer();
+        Boolean status = customersPage.addCustomer("Anoop Joseph","Flexible pouch manufacturer");
+        Assert.assertTrue(status);
+    }
+
+    //BATCH29-13
+    @Test
+    public void verifyNoOfRecords()
+    {
+        LoginPage loginPage = new LoginPage(driver);
+        loginPage.login("Admin","admin123",true,null);
+        CustomersPage customersPage = new CustomersPage(driver);
+        customersPage.navigateToCustomer();
+        Assert.assertEquals(customersPage.getNumberOfRecords(),customersPage.getActualNumberOfRecords());
+    }
+
+    //BATCH29-14
+    @Test
+    public void verifyNoOfRecordsAfterAddingCustomer()
+    {
+        LoginPage loginPage = new LoginPage(driver);
+        loginPage.login("Admin","admin123",true,null);
+        CustomersPage customersPage = new CustomersPage(driver);
+        customersPage.navigateToCustomer();
+        int DisplayedNumber1 = customersPage.getNumberOfRecords();
+        customersPage.addCustomer("Deepu Joseph","Web developer");
```

Comment:
To have the data in variables

Reply:
Created separate varibles to store the information